### PR TITLE
gradle: Clean up

### PR DESCRIPTION
**Pull request by lorenzleutgeb**

### CODE REVIEW

File: pkgs/development/tools/build-managers/gradle/default.nix [hunk 1]
```diff
@@ -57,28 +53,21 @@ rec {
         between the flexibility of Ant and the convenience of a
         build-by-convention behavior.
       '';
-      homepage = "http://www.gradle.org/";
+      homepage = "https://docs.gradle.org/${version}/";
```
Comment (kevincox) [inline pkgs/development/tools/build-managers/gradle/default.nix:56]: Is this standard? I found that most just go to the project homepage rather than docs.

Comment (kevincox) [reply pkgs/development/tools/build-managers/gradle/default.nix:56]: I would prefer to keep it as-is for now. Maybe we need a `docsPage` in the future. The other changes look good.

Comment (lorenzleutgeb) [reply pkgs/development/tools/build-managers/gradle/default.nix:56]: OK, reverting.

Comment (lorenzleutgeb) [reply pkgs/development/tools/build-managers/gradle/default.nix]: Fixed in 5153a6cf192

**Pull request merged**